frontend/src/components/pages/DashboardPage.tsx

```diff
@@ -28,7 +29,7 @@ const DashboardPage = (): React.ReactElement => {
 
     async function getUserDonations() {
         try {
-            const response = await axios.get('http://localhost:5000/donations/1');
+            const response = await axios.get('http://localhost:5001/donations/cly144mky0000bntg3dupxlx1');
```

Comment:
Can you pass in the frontend url from the env file instead of hardcoding it here?

That way, when we change the frontend url in the env file, this dashboard page will call the updated frontend url.